<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountFindDialog</class>
 <widget class="QDialog" name="accountFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="debitTransactionBalance">
     <property name="text">
      <string>Transaction Balance: $</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="debitAtmBalance">
     <property name="text">
      <string>ATM Withdrawal Balance: $</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="moneyTransactionFee">
     <property name="text">
      <string>Transaction Fee: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="creditGracePeriod">
     <property name="text">
      <string>Grace Period: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="creditAnnualRate">
     <property name="text">
      <string>Annual Percentage Rate: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="savingsInterestRate">
     <property name="text">
      <string>Interest Rate: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="savingsMinimumBalance">
     <property name="text">
      <string>Minimum Balance: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="certificateDepositTerm">
     <property name="text">
      <string>Deposit Term: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="certificateWithdrawalPenalty">
     <property name="text">
      <string>Withdrawal Penalty: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="moneyAllowedTransactions">
     <property name="text">
      <string>Allowed Transactions: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="accountFindType">
     <property name="text">
      <string>Type: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="accountFindBalance">
     <property name="text">
      <string>Balance: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="accountFindIban">
     <property name="text">
      <string>IBAN: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="accountFindOwner">
     <property name="text">
      <string>Owner PIN: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="checkingMonthlyFee">
     <property name="text">
      <string>Monthly Fee: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QDialogButtonBox" name="accountFindBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="debitAtmLimit">
     <property name="text">
      <string>ATM Withdrawal Limit: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="debitTransactionLimit">
     <property name="text">
      <string>Transaction Limit: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="moneyCurrentTransactions">
     <property name="text">
      <string>Current Transactions: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>accountFindBox</sender>
   <signal>accepted()</signal>
   <receiver>accountFindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accountFindBox</sender>
   <signal>rejected()</signal>
   <receiver>accountFindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
